cli_tools/common/image/importer/inflater.go

```diff
@@ -230,7 +233,18 @@ func newInflater(request ImageImportRequest, computeClient daisyCompute.Client,
 		return di, nil
 	}
 
-	ai := createAPIInflater(request, computeClient, storageClient, logger)
+	deadline, cancelFunc := context.WithDeadline(context.Background(), time.Now().Add(inspectionTimeout))
```

Comment:
This will cause inspection to run twice -- once here, and once in the daisy inflater. Can you run it once, and share the results?

Reply:
It's a bit tricky - once we switched off shadow test, we want to go back to the current logic - only run inspection for daisy inflater. Plus, I guess we will need the shadow test every time when PD team release a new format. So let's not simply remove the inspection logic from daisy inflater; instead, use a condition to determine whether to run it.